HookTest/units.py

```diff
@@ -390,32 +455,39 @@ def unique_passage(self):
         except Exception:
             yield False
 
-
     def has_urn(self):
         """ Test that a file has its urn saved
         """
         if self.xml is not None:
             if self.scheme == "tei":
-                urns = self.xml.xpath("//tei:text[starts-with(@n, 'urn:cts:')]", namespaces=TESTUnit.NS)
+                urns = self.xml.xpath("//tei:text/tei:body[starts-with(@n, 'urn:cts:')]", namespaces=TESTUnit.NS)
```

Comment:
Why this change here? Did we change the guidelines for this? Doesn't the URN belong on the text element?